docs/source/conf.py

```diff
@@ -64,7 +65,13 @@
 templates_path = ['_templates']
 
 # always execute notebooks
-nbsphinx_execute = 'always'
+env_skip_execute = os.getenv("SKIP_EXECUTE")
+
+if env_skip_execute:
```

Comment:
I got the condition backward
```suggestion
if not env_skip_execute:
```